docs/source/quickstart.mdx

```diff
@@ -0,0 +1,405 @@
+
+To compose our supergraph schema, we need the following information about each of our subgraphs:
+
+* The subgraph's schema
+* The URL of the subgraph's GraphQL endpoint (which must be accessible by the gateway)
+
+We'll start with just two subgraphs. Because we're using Apollo-hosted example services for our subgraphs, we know their endpoint URLs. We _don't_ have their schemas, however.
+
+#### Fetching subgraph schemas
+
+We can fetch a running subgraph's schema by executing an enhanced introspection query on it. The Rover CLI provides a command to do exactly that.
+
+From your project directory, run the following command in your terminal:
+
+```shell
+rover subgraph introspect https://7bssbnldib.execute-api.us-east-1.amazonaws.com/Prod/graphql
+```
+
+Rover introspects Apollo's example `products` service and outputs the following schema:
+
+<ExpansionPanel title="Click to expand">
+
+```graphql
+enum CURRENCY_CODE {
+  USD
+}
+
+type Department {
+  category: ProductCategory
+  url: String
+}
+
+type Money {
+  amount: Float
+  currencyCode: CURRENCY_CODE
+}
+
+"""Here are some helpful details about your type"""
+type Price {
+  cost: Money
+
+  """A number between 0 and 1 signifying the % discount"""
+  deal: Float
+  dealSavings: Money
+}
+
+"""
+This is an Entity, docs:https://www.apollographql.com/docs/federation/entities/
+You will need to define a __resolveReference resolver for the type you define, docs: https://www.apollographql.com/docs/federation/entities/#resolving
+"""
+type Product @key(fields: "id") {
+  id: ID!
+  title: String
+  url: String
+  description: String
+  price: Price
+  salesRank(category: ProductCategory = ALL): Int
+  salesRankOverall: Int
+  salesRankInCategory: Int
+  category: ProductCategory
+  images(size: Int = 1000): [String]
+  primaryImage(size: Int = 1000): String
+}
+
+enum ProductCategory {
+  ALL
+  GIFT_CARDS
+  ELECTRONICS
+  CAMERA_N_PHOTO
+  VIDEO_GAMES
+  BOOKS
+  CLOTHING
+}
+
+extend type Query {
+  bestSellers(category: ProductCategory = ALL): [Product]
+  categories: [Department]
+  product(id: ID!): Product
+}
+```
+
+</ExpansionPanel>
+
+Now run that same command, but this time append ` > products.graphql`:
+
+```shell
+rover subgraph introspect https://7bssbnldib.execute-api.us-east-1.amazonaws.com/Prod/graphql > products.graphql
+```
+
+This writes the subgraph's schema to a file instead of printing it to your terminal.
+
+Next, do the same thing for the _second_ subgraph (`reviews`), this time writing it to `reviews.graphql`:
+
+```shell
+rover subgraph introspect https://w0jtezo2pa.execute-api.us-east-1.amazonaws.com/Prod/graphql > reviews.graphql
+```
+
+Great! We now have the schemas for our two subgraphs.
+
+### Providing subgraph details
+
+After we obtain our subgraph schemas and URLs, we need to provide them to the Rover CLI. To do that, we add them to a YAML file in our project directory.
+
+Create a file called `supergraph.yaml` and paste the following into it:
```

Comment:
NGL the fact that this file was named 'supergraph' was very confusing to me given that there's also supergraph.graphql, and this supergraph.yaml is not a supergraph schema and is just a config file for this project.

Reply:
Renaming to `supergraph-config.yaml`